### PR TITLE
E-720 fix: move scopes and redirects to controller

### PR DESCRIPTION
**Pull request by dudiq**

### CODE REVIEW

**Comment by dudiq** [comment]
@lamartire ready for review

File: src/components/common/VError.vue [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <template>
   <div>
     <message :error="true">
-      {{ hint }}
+      {{ label }}
```
Comment (dudiq) [inline src/components/common/VError.vue]: comment here

File: src/components/screens/public/ConsentProvider/ConsentProviderController.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+  constructor(props, { accountsStore = accountsStoreModule }) {
+    super(props);
+    this.accountsStore = accountsStore;
+  }
+
+  openRedirectUrl(url) {
+    window.location.href = url;
+    setTimeout(() => {
+      this.error = i18n.t('components.consentProvider.redirectUrl');
```
Comment (lamartire) [inline src/components/screens/public/ConsentProvider/ConsentProviderController.js]: Is this error related to timeout exception? If yes - not obvious

File: src/components/screens/public/ConsentProvider/ConsentProviderController.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    }, OPEN_URL_TIMEOUT_MS);
+  }
+
+  /**
+   * @param {object} params
+   * @param {String[]} params.scopesList
```
Comment (lamartire) [inline src/components/screens/public/ConsentProvider/ConsentProviderController.js]: `string[]`

File: src/components/screens/public/ConsentProvider/ConsentProviderController.js (hunk 2 above)
Comment (dudiq) [reply src/components/screens/public/ConsentProvider/ConsentProviderController.js]: yes, error related to timeout.

how to explain by other methods for make it more obvious?

File: src/components/screens/public/ConsentProvider/ConsentProviderController.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+    window.location.href = url;
+  }
+
+  /**
+   * @param {object} params
+   * @param {string[]} params.scopesList
+   * @return {Promise<void>}
+   */
+  @Action
+  async submitScopes({ consentChallenge, scopesList }) {
```
Comment (kazurus) [inline src/components/screens/public/ConsentProvider/ConsentProviderController.js]: grantScopes

File: src/components/screens/public/ConsentProvider/ConsentProviderController.js [hunk 5]
```diff
@@ -0,0 +1,84 @@
+import { VuexModule, Action, Module } from 'vuex-class-modules';
+import createController from '@/controllers/createController';
+import { accountsStore as accountsStoreModule } from '@/store';
+import i18n from '@/locales/i18n';
+
+const OPEN_URL_TIMEOUT_MS = 5000;
+
+@Module({ generateMutationSetters: true })
+class ConsentProviderController extends VuexModule {
+  error = '';
```
Comment (kazurus) [inline src/components/screens/public/ConsentProvider/ConsentProviderController.js]: it's not a good idea to save error in controller. Can we move error to component?

File: src/components/screens/public/ConsentProvider/ConsentProviderController.js (hunk 2 above)
Comment (lamartire) [reply src/components/screens/public/ConsentProvider/ConsentProviderController.js]: `components.consentProvider.redirectTimeoutError` I guess

**Comment by ifiri** [comment]
@kazurus seems to all is ok. Approved

File: src/components/screens/public/ConsentProvider/ConsentProviderController.js (hunk 5 above)
Comment (dudiq) [reply src/components/screens/public/ConsentProvider/ConsentProviderController.js]: have one idea, will try to implement

Comment (dudiq) [reply src/components/screens/public/ConsentProvider/ConsentProviderController.js]: @lamartire @kazurus @ifiri 
updated approach with set error message
